src/lio/bin/lio_mv.c

```diff
@@ -297,6 +297,7 @@ int main(int argc, char **argv)
         while ((gop = opque_get_next_failed(q)) != NULL) {
             j = gop_get_myid(gop);
             info_printf(lio_ifd, 0, "Failed with path %s\n", flist[j].src_tuple.path);
+            status = gop_get_status(gop);
             n_errors += status.error_code;
```

Comment:
And that's what it means although you only care if it's non-zero. There 
needs to be code cleanup before the main loop since some of 
fail-on-recovery logic was absorbed in lower level routines.